src/components/customMdx/admonition.tsx

```diff
@@ -16,20 +16,39 @@ const colorMap: any = {
 }
 
 const Admonition = ({ children, type, ...props }: AdmonitionProps) => {
+  console.log({children})
+  const isMultiline: boolean = Array.isArray(children) ? true : false;
```

Comment:
```suggestion
  const isMultiline: boolean = Array.isArray(children) ? true : false;
```
I think we should remove the console.log?

Reply:
![face-palm](https://user-images.githubusercontent.com/22930449/129033895-4a1101cc-6e11-42d2-afff-6ffa26ccf1a6.gif)